pillar/flavors/standard.sls

```diff
@@ -45,6 +45,9 @@
             "parallelism": 1,
             "taskmanager_mem_preallocate": false,
             "pyflink_yarn_container_count": 1
+        },
+        "graphite-api":{
+            "retentions": "60:10080"
```

Comment:
Standard should have a retention between that of production and pico.

Reply:
For standard, have set retention 7 days.